pkgs/native_toolchain_c/lib/src/cbuilder/run_cbuilder.dart

```diff
@@ -246,6 +250,10 @@ class RunCBuilder {
           if (value == null) '-D$name' else '-D$name=$value',
         for (final include in includes) '-I${include.toFilePath()}',
         ...sourceFiles,
+        if (language == Language.objectiveC) ...[
+          '-framework',
+          'Foundation',
```

Comment:
It might be nicer to add a `frameworks` option, like the `includes` option, and default it to `['Foundation']`.

Reply:
Done.